### DISCUSSION TITLE
delayed in showing mail on browser

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#370] [Q&A]
Hi,
I am using inbucket for my organisation. when i send mail, it took 30-50 second to appear on web browser. can you tell me what shoud i do to make mail delivery fast

**Comment by jhillyerd** [comment]
First step would be to determine whether it's slow delivery to inbucket, or inbucket being slow to update by examining the logs:

set log level to `info`: https://github.com/inbucket/inbucket/blob/main/etc/linux/inbucket.service#L11

as root:

```
systemctl restart inbucket.service
journalctl -u inbucket.service --follow
```
